src/platform/suecreek/include/platform/lib/memory.h

```diff
@@ -402,7 +402,9 @@
 #define BOOT_LDR_STACK_SIZE		SOF_STACK_TOTAL_SIZE
 
 /* code loader entry point for base fw */
-#define SRAM_VECBASE_RESET	(BOOT_LDR_BSS_BASE + BOOT_LDR_BSS_SIZE)
+#define _SRAM_VECBASE_RESET (BOOT_LDR_BSS_BASE + BOOT_LDR_BSS_SIZE)
+#define SRAM_VECBASE_RESET ((_SRAM_VECBASE_RESET) - ((_SRAM_VECBASE_RESET) % \
```

Comment:
This is in another PR

Reply:
I assume it's required for testing hence it's also included here.

Reply:
@lgirdwood @paulstelian97 
Yes, I will remove it. It is only to calm down travis.